pilot/pkg/model/authorization.go

```diff
@@ -35,17 +37,21 @@ type RolesAndBindings struct {
 
 // AuthorizationPolicies organizes authorization policies by namespace.
 type AuthorizationPolicies struct {
-	// Maps from namespace to RolesAndBindings.
-	namespaceToV1Policies map[string]*RolesAndBindings
+	// Maps from namespace to the v1alpha1 RBAC policies, deprecated by v1beta1 Authorization policy.
+	namespaceToV1alpha1Policies map[string]*RolesAndBindings
 
-	// The mesh global RbacConfig.
+	// The mesh global RbacConfig, deprecated by v1beta1 Authorization policy.
 	rbacConfig *rbacproto.RbacConfig
+
+	// Maps from namespace to the v1beta1 Authorization policies.
+	namespaceToV1beta1Policies map[string][]Config
```

Comment:
Just curious, why do you want to keep the Config here instead of the policy proto ?

Reply:
Some metadata (e.g. name) from the Config would be useful when generating the Envoy filter config for letting us know where it's originated from.

Reply:
I'm not good at golang object. Do you know this will be a copy or just a pointer?

Reply:
The value is a slice (`[]Config`) which consists of a pointer, a length int and a capacity int, in other words, here it can just be treated like a pointer (not copying the Config).

Reply:
Oh, maybe you're asking for `Config`, the `Spec` field is an interface which I believe is just similar to pointer.